docs/releasehistory.rst

```diff
@@ -7,6 +7,78 @@ Releases follow the ``major.minor.micro`` scheme recommended by `PEP440 <https:/
+The spec update is the result of discussions about how to handle the evolution of data and parameter types as further functional forms are added to the SMIRNOFF spec.
+
+
+We provide methods to convert SMIRNOFF 0.1 and 0.2 forcefields written with the XML serialization (.offxml) to the SMIRNOFF 0.3 XML specification.
+These methods are called automatically when loading a serialized SMIRNOFF data representation written in the 0.1 or 0.2 specification.
+This functionality allows the toolkit to continue to read files containing SMIRNOFF 0.2 spec force fields, and also implements backwards-compatibility for SMIRNOFF 0.1 spec force fields.
+
+
+WARNING: The SMIRNOFF 0.1 spec did not contain fields for several energy-determining parameters that are exposed in later SMIRNOFF specs.
+Thus, when reading SMIRNOFF 0.1 spec data, the toolkit must make assumptions about the values that should be added for the newly-required fields.
+The values that are added include 1-2, 1-3 and 1-5 scaling factors, cutoffs, and long-range treatments for nonbonded interactions.
+Each assumption is printed as a warning during the conversion process.
+Please carefully review the warning messages to ensure that the conversion is providing your desired behavior.
+
+
+
+`SMIRNOFF 0.3 specification updates <https://open-forcefield-toolkit.readthedocs.io/en/0.4.0/smirnoff.html>`_
+"""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+* The SMIRNOFF 0.3 spec introduces versioning for each individual parameter section, allowing asynchronous updates to the features of each parameter class.
+  The top-level ``SMIRNOFF`` tag, containing information like ``aromaticity_model``, ``Author``, and ``Date``, still has a version (currently 0.3).
+  But, to allow for independent development of individual parameter types, each section (such as ``Bonds``, ``Angles``, etc) now has its own version as well (currently all 0.3).
+* All units are now stored in expressions with their corresponding values. For example, distances are now stored as ``1.526*angstrom``, instead of storing the unit separately in the section header.
+* The ``potential`` field for ``ProperTorsions`` and ``ImproperTorsions`` tags is no longer ``charmm``, but is rather ``k*(1+cos(periodicity*theta-phase))``.
+  It was pointed out to us that CHARMM-style torsions deviate from this formula when the periodicity of a torsion term is 0, and we do not intend to reproduce that behavior.
+* SMIRNOFF spec documentation has been updated with tables of keywords and their defaults for each parameter section and parameter type.
+  These tables will track the allowed keywords and default behavior as updated versions of individual parameter sections are released.
+
+Performance improvements
+""""""""""""""""""""""""
+
+* `PR #329 <https://github.com/openforcefield/openforcefield/pull/329>`_: Improved runtime for system creation in large topologies.
```

Comment:
"Improved runtime" -> "Performance improvements". And does "large" refer to many atoms or many molecules?

Reply:
"large" refers to "many atoms". Changed to:

`Performance improvements when creating systems for topologies with many atoms.`